stories/Content/Tag/RemovableTag/RemovableTag.stories.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES, TAG_COLORS } from "../../confetti-ds/src/constants";
-import { Button, RemovableTag as Component } from "../../confetti-ds/src";
+import { ICON_TYPES, TAG_COLORS } from "../../../../confetti-ds/src/constants";
+import { Button, RemovableTag as Component } from "../../../../confetti-ds/src";
 
 export default {
-  title: "Tag/Removable Tag",
+  title: "Content/Tag/Removable Tag/Removable Tag",
```

Comment:
```suggestion
  title: "Content/Tag/Removable Tag",
```

Removable Tag stories from Tag QA should use this title too.

Reply:
fixed